DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests.cs

```diff
@@ -240,5 +240,38 @@ public void UpdateDelegateUserCentrePrompts_updates_user()
                 }
             }
         }
+
+        [Test]
+        public void ApproveDelegateUsers_approves_delegate_users()
+        {
+            using (var transaction = new TransactionScope())
+            {
+                try
+                {
+                    // Given
+                    var ids = new[] { 16, 28 };
+                    var delegate1 = userDataService.GetDelegateUserById(16);
+                    var delegate2 = userDataService.GetDelegateUserById(28);
+
+                    delegate1.Approved.Should().BeFalse();
+                    delegate2.Approved.Should().BeFalse();
+
+                    // When
+
+                    userDataService.ApproveDelegateUsers(ids);
+
+                    delegate1 = userDataService.GetDelegateUserById(16);
+                    delegate2 = userDataService.GetDelegateUserById(28);

```

Comment:
Instead of overriding delegate1 & delegate2, it might be worth declaring new vars, e.g. delegate1AfterUpdate or something similar (overriding variables can cause confusion at times)